### PR TITLE
pkg/logcli/query: clean DoQuery

### PR DESCRIPTION
**Pull request by dvrkps**

### CODE REVIEW

**Comment by slim-bean** [comment]
Hello @dvrkps, can you add some comments here with what you are looking to accomplish with this PR?

This particular code section is complex, i'm reluctant to change it without a little explanation.

**Comment by dvrkps** [comment]
Hi @slim-bean, i want to make code more idiomatic by keep happy path more left.

File: pkg/logcli/query/query.go [hunk 1]
```diff
@@ -80,76 +79,76 @@ func (q *Query) DoQuery(c client.Client, out output.LogOutput, statistics bool)
...
+		return
+	}
 
-			if statistics {
-				q.printStats(resp.Data.Statistics)
-			}
+	if q.Limit < q.BatchSize {
+		q.BatchSize = q.Limit
+	}
+	resultLength := q.BatchSize
```
Comment (cyriltovena) [inline pkg/logcli/query/query.go:88]: I think this was inefficient assignment.
```suggestion
	resultLength := 0
```

**Comment by cyriltovena** [comment]
@dvrkps I agree with you the code is not very idiomatic. @slim-bean this is just some improvement on `if` `else` branching, I think it looks good. WDYT ?

File: pkg/logcli/query/query.go (hunk 1 above)
Comment (dvrkps) [reply pkg/logcli/query/query.go:88]: @cyriltovena is resultLength in line 88 depends to q.BatchSize in line 86.

**Pull request closed without merging**